pkg/utils/dryrun/dryrun.go

```diff
@@ -0,0 +1,141 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dryrun
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"strings"
+	"sync"
+
+	"sigs.k8s.io/kwok/pkg/utils/path"
+	"sigs.k8s.io/kwok/pkg/utils/slices"
+)
+
+// DryRun is a flag to indicate whether the program is running in dry-run mode.
+var DryRun bool
+
+// IsDryRun returns true if the program is running in dry-run mode.
+func IsDryRun() bool {
+	return DryRun
+}
```

Comment:
Should we move these to pkg/config/vars.go? Considering dryrun is also a config var of kwokctl like others.

Reply:
emmm, It should not be suitable, `pkg/config/vars.go` is for `kwok`/`kwokctl`, `dryrun` is only for `kwokctl` and contains some additional output functions that do not correspond to the config.

Reply:
I mean only move the DryRun variable definition and getter function to the config package.  Putting these additional output functions in an independent util package looks good to me. IMO, it's a bit odd to set and get a config value from a util package.

Reply:
> it's a bit odd to set and get a config value from a util package.
That makes sense, then I think we can move dryrun,file,exec to `pkg/kwokctl`.